client-oauth-discovery-dui/pom.xml

```diff
@@ -79,6 +79,16 @@
       <version>3.1.0</version>
       <scope>provided</scope>
     </dependency>
+	<dependency>
+		<groupId>javax.servlet</groupId>
+		<artifactId>jstl</artifactId>
+		<version>1.2</version>
+	 </dependency>
+	 <dependency>
+	    <groupId>taglibs</groupId>
+	    <artifactId>standard</artifactId>
```

Comment:
AFAIK, taglibs:standard is the same thing as jstl or jstl-impl. Can we avoid pulling it too?